packages/core/src/renderer/components/+workloads-pods/load-pods-from-all-namespaces.injectable.ts

```diff
@@ -4,20 +4,20 @@
  */
 
 import { getInjectable } from "@ogre-tools/injectable";
-import namespaceStoreInjectable from "../+namespaces/store.injectable";
+import clusterFrameContextForNamespacedResourcesInjectable from "../../cluster-frame-context/for-namespaced-resources.injectable";
 import showErrorNotificationInjectable from "../notifications/show-error-notification.injectable";
 import podStoreInjectable from "./store.injectable";
 
 const loadPodsFromAllNamespacesInjectable = getInjectable({
   id: "load-pods-from-all-namespaces",
   instantiate: (di) => {
     const podStore = di.inject(podStoreInjectable);
-    const namespaceStore = di.inject(namespaceStoreInjectable);
+    const context = di.inject(clusterFrameContextForNamespacedResourcesInjectable);
     const showErrorNotification = di.inject(showErrorNotificationInjectable);
-  
+
     return () => {
       podStore.loadAll({
-        namespaces: [...namespaceStore.getItems().map(ns => ns.getName())],
+        namespaces: context.allNamespaces,
```

Comment:
Why do we need to pass `namespaces` here, shouldn't it default to all namespaces if none given?